app/src/src-SessionBear/App.js

```diff
@@ -32,6 +33,7 @@ const App = () => {
 
   useIsExtensionEnabled();
   submitAppDetailAttributes();
+  useInitializeNewUserSessionRecordingConfig();
```

Comment:
the compatible extension version in here is `23.10.22`
make sure to verify if that is correct